apps/src/sites/studio/pages/weblab_host/test.js

```diff
@@ -0,0 +1,14 @@
+import $ from 'jquery';
+import React from 'react';
+import ReactDOM from 'react-dom';
+import WebLabTest from '@cdo/apps/templates/testPages/WebLabTest';
```

Comment:
One thought: file names that include "test" almost always refer to unit/UI/integration tests. There's been some confusion in the past around the files that are not those types of tests but have test in the name. Would something like "WebLabSupportVerification" or similar communicate the same intent without leading to that confusion?

Reply:
That seems fine to me -- Brendan and I chatted about this briefly, but I couldn't come up with an alternate name I was happy with. `WebLabSupportVerification` seems good. Are we also satisfied with that language in the URL, i.e., `studio.code.org/weblab/test` to `studio.code.org/weblab/support-verification`?

Reply:
Oh huh. That would be confusing. I could see something like /weblab/network-check instead. (That being said, I generally recommend avoiding using "test" except if there is a good reason. Clarity in a user-facing scenario is a good reason.)

Reply:
`/network-check` is probably clearer/more descriptive than `/test` to users too, right? Are we happy that all of the things we are testing actually fall under the "network" umbrella? (I think you could make an argument that they do, and I'm not sure that an alternative like `/system-check` is actually broader.)

Reply:
We can make the user-facing URL whatever we want if we're happy with one line of code that does `/weblab/network-check` -> `weblab_host#support_verification.` I like that using "support verification" internally is descriptive enough to hold space for any/all other potential test pages we could create.